rest/bulk_api.go

```diff
@@ -262,7 +262,7 @@ func (h *handler) handleDump() error {
 
 // HTTP handler for _repair
 func (h *handler) handleRepair() error {
-
+	base.Audit(h.ctx(), base.AuditIDDatabaseRepair, nil)
```

Comment:
I don't believe this should be emitting audit logs (since this endpoint is unused and always returns error).  If the goal is to have it in place so it's not missed if the endpoint gets enabled in future, maybe switch to something like:
```
err := errors.New("_repair endpoint disabled")
if err != nil {
  base.Audit(h.ctx(), base.AuditIDDatabaseRepair, nil)
}
return err
```